app/code/Magento/Checkout/view/frontend/web/template/minicart/content.html

```diff
@@ -25,7 +25,8 @@
                 attr: {
                     title: $t('Close')
                 },
-                click: closeMinicart()
```

Comment:
@rogerdz 
I think that changing `click: closeMinicart()` to `click: closeMinicart` would be enough here